src/components/AdminPortal/EventRegistrantsModal/Modal/AddOnSpot/AddOnSpotSuccessModal.tsx

```
@@ -0,0 +1,123 @@
+ *   handleClose={() => setShow(false)}
+ * />
+ * ```
+ *
+ * Dependencies
+ * - shared-components/BaseModal
+ * - NotificationToast
+ * - react-i18next
+ * - react-bootstrap Button
+ */
+
+import { Button } from 'react-bootstrap';
```

ESLint

[error] 'Button' import from 'react-bootstrap' is restricted. Direct imports of Button from react-bootstrap are not allowed. Use the shared Button component from src/shared-components/Button/ instead.

(no-restricted-imports)